apps/web/src/custom-pages/weekly-newsletter-page.tsx

```diff
@@ -28,12 +28,27 @@ function NewsItemContent({ content }: { content?: EditorState }) {
   );
 }
 
-function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+async function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+  const t = await getScopedI18n("weeklyNewsletter");
   const content = newsItem.content as unknown as EditorState;
 
   return (
     <article>
       <h3 id={stringToId(newsItem.title)}>{newsItem.title}</h3>
+      {newsItem.linkToSignUp ? (
+        <a
```

Comment:
I think we should use Link element from `next/link`, so we get fancy optimization and prefetch